cfme/tests/cloud/test_azure_manual.py

```diff
@@ -0,0 +1,241 @@
+import pytest
+
+pytestmark = [pytest.mark.manual]
+
+
+@pytest.mark.manual
+def test_vpc_env_selection():
+    """
+    Test selection of components in environment page of cloud instances
+    with and without selected virtual private cloud
+    Related to BZ 1315945
```

Comment:
Please use `Bugzilla:` header in docblock, or `pytest.mark.meta(automates=[BZ(number)])` so that this metadata information can be programatically accessed.

Reply:
Ok, I'm adding in a docblock, as it's a manual test and `automates` doesn't fit here (in my opinion)